resources/zib-HealthProfessional-Practitioner.xml

```diff
@@ -38,7 +38,7 @@
       <path value="Practitioner" />
       <short value="HealthProfessional" />
       <definition value="A health professional is a person who is authorized to perform actions in the field of individual healthcare." />
-      <comment value="The zib HealthProfessional is mapped for all but one concept (HealthProfessionalRole) to Practitioner and PractitioneRole. PractitionerRole covers the recording of the location and types of services that HealthProfessionals are able to provide for a HealthcareProvider. The zib concepts 'HealthProfessionalIdentificationNumber', 'Specialty' and 'HealthcareProvider' are therefore mapped to PractitionerRole. &#xD;&#xA;&#xD;&#xA;Although the zib concept name HealthProfessionalRole feels a match for PractitionerRole, this is not the case based on the definition and values of that concept. In contrary to the zib, the more clinical resources contain the role information the Healthprofessinal fullfills in the that specific case. An example is the `Procedure.performer.role element`. This prevents creating many HealthProfessional instances." />
+      <comment value="The zib HealthProfessional is mapped for all but one concept (HealthProfessionalRole) to this Practitioner profile and a profile on PractitioneRole (&lt;http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-PractitionerRole&gt;). The PractitionerRole resource covers the recording of the location and types of services that HealthProfessionals are able to provide for a HealthcareProvider. The zib concepts 'HealthProfessionalIdentificationNumber', 'Specialty' and 'HealthcareProvider' are therefore mapped onto PractitionerRole.&#xD;&#xA;&#xD;&#xA;Although the zib concept name 'HealthProfessionalRole' superficially seems to match the PractitionerRole resource, this is not the case based on the definition and values of that concept. In FHIR, the role that a HealthProfessional fulfills is captured as part as the clinical resource for that specific case, for example in the `Procedure.performer.role` element. This prevents the creation of duplicate HealthProfessional instances for each clinical situation where the HealthProfessional performs a different role in the process." />

```

Comment:
Goede toevoeging. Misschien kunnen de quotjes ('') ook wel weg bij de zib conceptnamen. Dat doen we nergens anders ook echt. Wat denk jij?

Reply:
Goed punt, laten we het wel consistent doen.

Reply:
Verwerkt.